wire-ios/Wire-iOS/Sources/UserInterface/UserProfile/ProfileActionsFactory.swift

```diff
@@ -150,7 +150,7 @@ final class ProfileActionsFactory {
         }
 
         Task {
-            let isOneOnOneReady = try await userSession.checkOneOnOneConversationIsReady.invoke(userID: userID)
+            let isOneOnOneReady = await isOneOnOneReady(userID: userID)
```

Comment:
@caldrian good example for the SwiftLint rule [`unhandled_throwing_task`](https://realm.github.io/SwiftLint/unhandled_throwing_task.html)? Would it have prevented that?